deployments/common/crds/k8s.nginx.org_virtualservers.yaml

```diff
@@ -479,6 +479,26 @@ spec:
                   description: TLS defines TLS configuration for a VirtualServer.
                   type: object
                   properties:
+                    cert-manager:
+                      description: CertManager defines a cert manager config for a TLS.
```

Comment:
I think descriptions should only refer to themselves as they are named in the schema. `cert-manager` defines yada yada.

Looks like a pattern across the schema definition, has anyone brought this up as an issue?

Reply:
Good point, I think for now I'd rather leave it consistent with what is there, and raise this as a discussion with the team.